tests/scripts/thread-cert/Cert_5_3_06_RouterIdMask.py

```diff
@@ -127,7 +129,9 @@ def test(self):
         self.nodes[ROUTER2].start()
         time.sleep(5)
         self.assertEqual(self.nodes[ROUTER2].get_state(), 'router')
-        time.sleep(64)
+
+        # It might take MAX_ADVERTISEMENT_INTERVAL * 2 to transmit advertisement along ROUTER2-->ROUTER1-->DUT_LEADER.
+        time.sleep(config.MAX_ADVERTISEMENT_INTERVAL * 2)
```

Comment:
I don't think it is necessary to wait so long here. You are not validating the cost to ROUTER2 afterwards, it should be fine to only ensure ROUTER2 is an active router, that is, above line 131 is enough.